libs/darts/darts.ts

```diff
@@ -93,7 +94,7 @@ class Dart extends sprites.ExtendableSprite {
     //% weight=8
     public angleRate: number;
     //% group="Properties" blockSetVariable="myDart"
-    //% blockCombine block="wind"
+    //% blockCombine block="power adjust rate"
```

Comment:
What does this change do?

Reply:
This is the block name for `powerAdjust`. The rate at which the power value is changed as your press the up/down arrows.